src/protagonist/API/Features/Customer/Requests/CreatePortalUser.cs

```diff
@@ -57,9 +57,11 @@ public async Task<CreatePortalUserResult> Handle(CreatePortalUser request, Cance
             return new CreatePortalUserResult { Error = "Email address is invalid" };
         }
 
-        var userWithEmail = await dbContext.Users.SingleOrDefaultAsync(
-            u => u.Email == request.PortalUser.Email, cancellationToken: cancellationToken);
-        if (userWithEmail != null)
+        // get all locally for more string comparison support
+        var allUsers = await dbContext.Users.ToListAsync(cancellationToken);
```

Comment:
This is inefficient - not ideal but `.toLower()` both should cover this use case. A better long term solution would be to sort the collation used in DB

Reply:
Will change. I think it's also worth mentioning that this is currently how it's handled in `CreateCustomer`
https://github.com/dlcs/protagonist/blob/7ea6586c51dac5ed21ecc8dc1bfd099f61446682/src/protagonist/API/Features/Customer/Requests/CreateCustomer.cs#L137